config/etc/system-image/writable-paths

```diff
@@ -13,6 +13,7 @@
 /tmp                                    none                    temporary   none        defaults
 /mnt                                    none                    temporary   none        defaults
 /media                                  none                    temporary   none        defaults
+/boot/androidboot                       auto                    persistent  none        none
```

Comment:
Can you explain a bit more why we need this to be a writable-path? AFAIK we have /boot/grub and /boot/uboot on a dedicated boot partition so far. Adding /boot/androidboot here would mean we have /boot/androidboot on every device running Ubuntu Core unless I miss something.